ansible/roles_ocp_workloads/ocp4_workload_cloud_native_robot/defaults/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+ocp4_workload_cloud_native_robot_user_count: >-
+  {{ ocp4_workload_authentication_htpasswd_user_count
+   | default(user_count)
+   | default(num_users)
+   | default(4)
+  }}
+ocp4_workload_cloud_native_robot_user_prefix: >-
+  {{ (ocp4_workload_authentication_htpasswd_user_base | default('team')) }}
+
+ocp4_workload_cloud_native_robot_infra_repo: https://github.com/cloud-native-robotz-hackathon/infrastructure.git
+ocp4_workload_cloud_native_robot_infra_repo_tag: main
```

Comment:
is there a pinned version that we can use for provision?

(part of global policy for external dependencies, prefer  tag over moving targets like branch)

This can be overwritten in agnosticV but i think the default should be a tag otherwise it'll be hidden from the catalog maintainers.

Reply:
Good point. The idear is to handle this via agnostiv and make it change able: https://github.com/rhpds/agnosticv/blob/master/sandboxes-gpte/CLOUD_NATIVE_ROBOT/common.yaml#L330-L337

It's not perfect yet, because we are still under development.

Reply:
Good point, I provide that in dev: https://github.com/rhpds/agnosticv/pull/15684

For a final workshop, we will provide a dedicated tags. Thanks. For me, it's resolved.